test/Benchmarks/Metrics/HistogramBenchmarks.cs

```diff
@@ -83,7 +83,7 @@ public void Setup()
                 this.bounds[i] = i * MaxValue / this.bounds.Length;
             }
 
-            var exportedItems = new List<Metric>();
+            var exportedItems = new List<ExportableMetricCopy>();
```

Comment:
is it required to force every test to use the new MetricCopy unless they need to rely on it? 
Or is it by design that InMemoryExporter will force one to use the MetricCopy option only? What I don't need/care about the fact that Metric could be update after export() call is done?

Reply:
Yes, current approach was to force the use of the Copy.
See the comment above for more details https://github.com/open-telemetry/opentelemetry-dotnet/pull/3198#discussion_r867199870